react/components/EditorContainer/StoreEditor/Store/PWAForm/index.tsx

```diff
@@ -119,30 +132,40 @@ const PWAForm: React.FunctionComponent<Props> = ({
       ],
       manifest
     ) as ManifestMutationData
+    const result = await updateManifest.mutate({
+      variables: { manifest: manifestData },
+    })
+    return path(['data', 'updateManifest'], result || {})
+  }
 
+  async function handleSubmit() {
+    setSubmitting(true)
     try {
-      const result = await updateManifest.mutate({
-        variables: { manifest: manifestData },
-      })
-      const backgroundColor: string | undefined = path(
-        ['data', 'updateManifest', 'background_color'],
-        result || {}
-      )
+      const mutations: Array<Promise<any>> = [saveManifest()]
```

Comment:
Why the `any`, since `saveManifest` is correctly typed?

Reply:
`saveSettings` is typed differently, but I'll use the `|` operand